client/src/routes/home/Dashboard.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import withStyles from 'isomorphic-style-loader/withStyles';
 import s from './Home.css';
```

Comment:
Rename the style file as well please, for consistency

Reply:
other files also contain style files named like this, i thought it was some kind of pattern

Reply:
Other files also contain files? Paraphrase this, please

Reply:
for example, we have home route that imports style file like this `import s from './Home.css'`

Reply:
Probably we got each other wrong. I mean this line to be `import s from './Dashboard.css';`

Reply:
Ohh, yeah, now I understand, okay